rust/README.md

```diff
@@ -0,0 +1,7 @@
+# [Collatz](https://github.com/Skenvy/Collatz): [Rust](https://github.com/Skenvy/Collatz/tree/main/rust) ♥️🦀♥️
+Functions related to [the Collatz/Syracuse/3N+1 problem](https://en.wikipedia.org/wiki/Collatz_conjecture), implemented in [Rust](https://doc.rust-lang.org/reference/index.html).
```

Comment:
Can the link for Rust be https://www.rust-lang.org/ 
I'd probably mention the reference docs in the Devlog though!

Reply:
Updated this accordingly to reflect the https://www.rust-lang.org/ url